app/serializers/user_profile_serializer.rb

```diff
@@ -9,4 +9,13 @@ class UserProfileSerializer
   attribute :orders do |user|
     user.orders ? user.orders.map { |order| OrderSerializer.new(order).serializable_hash } : []
   end
+
+  # Added the profile_image_url method here
+  def profile_image_url
```

Comment:
object is not available directly in the UserProfileSerializer. 
attribute method should be used instead

```suggestion
  attribute :profile_image_url do |user|
```